snappi_ixnetwork/trafficitem.py

```diff
@@ -191,9 +220,245 @@ class TrafficItem(CustomField):
 
     def __init__(self, ixnetworkapi):
         self._api = ixnetworkapi
+        self.ixn_config = None
+        self.traffic_index = 1
         self.has_latency = False
         self._flow_timeout = 10
 
+    def _get_search_payload(self, parent, child, properties, filters):
+        payload = {
+            "selects": [
+                {
+                    "from": parent,
+                    "properties": [],
+                    "children": [
+                        {
+                            "child": child,
+                            "properties": properties,
+                            "filters": filters,
+                        }
+                    ],
+                    "inlines": [],
+                }
+            ]
+        }
+        url = "{}/operations/select?xpath=true".format(
+            self._api.assistant._ixnetwork.href
+        )
+        return (url, payload)
+
+    def _export_config(self):
+        href = "%sresourceManager" % self._api._ixnetwork.href
+        url = "%s/operations/exportconfig" % href
+        payload = {
+            "arg1": href,
+            "arg2": ["/traffic/trafficItem/descendant-or-self::*"],
+            "arg3": True,
+            "arg4": "json",
+        }
+        res = self._api._request("POST", url=url, payload=payload)
+        return json.loads(res["result"])
+
+    def _importconfig(self, imports):
+        imports["xpath"] = "/"
+        href = "%sresourceManager" % self._api._ixnetwork.href
+        url = "%s/operations/importconfig" % href
+        import json
+
+        payload = {
+            "arg1": href,
+            "arg2": json.dumps(imports),
+            "arg3": False,
+            # "arg4": "suppressNothing",
+            # "arg5": True,
+        }
+        try:
+            # TODO for larger config rest api is throwing error,
+            # with no url found, when the first response is 202 (in-progress)
+            # its keep checking the status of the url with 1 sec sleep, and
+            # after a while error is thrown. but could see the configuration
+            # applied at Ixnetwork. (Need to check with Eng team)
+            self._api._request("POST", url=url, payload=payload)
+        except Exception:
```

Comment:
I am not sure how we will handle as json import error are very generic. User also may not map IxNet internal object with sanppi object. Anyway, first put all error. Then we will decide.

Reply:
this was time being addition to make sure all the tests passes. once that is done. I will add the check if the exception raised after the 201[Inprogress] response. which needs a exception handling in snappi_api.py _requests method